.vscode/settings.json

```diff
@@ -1,6 +1 @@
-{
-  "workbench.colorCustomizations": {
-    "titleBar.activeBackground": "#f069c5",
-    "titleBar.activeForeground": "#000000"
```

Comment:
why remove this?

Reply:
You are right. It looks like my VS didn't like the color. Will change it back.